algorithms/src/polycommit/sonic_pc/mod.rs

```diff
@@ -128,9 +128,11 @@ impl<E: PairingEngine, S: AlgebraicSponge<E::Fq, 2>> SonicKZG10<E, S> {
             .map(|(_k, v)| *v)
             .collect::<Vec<_>>();
         if powers_of_beta_times_gamma_g.len() != supported_hiding_bound + 2 {
-            return Err(
-                PCError::HidingBoundToolarge { hiding_poly_degree: supported_hiding_bound, num_powers: 0 }.into()
-            );
+            return Err(anyhow::anyhow!(
```

Comment:
This syntax seems to work and is less verbose (the same applies to many places; perhaps a search of `return Err(anyhow` can find them all)
```suggestion
            anyhow::bail!(
```